README.md

```diff
@@ -23,31 +27,121 @@ const job = await client.createReport({
   endTime: new Date().toISOString(),
   browserName: "Chrome",
   browserVersion: "105",
-  framework: "playwright",
+  framework: "playwright", // alternatives: cypress, playwright, testcafe
   frameworkVersion: "1.25.0",
   platformName: "Windows 11"
 });
 console.log(job.id); // the job ID
 console.log(job.url); // the full URL of the job
 ```
 
-## Upload Assets
+## Upload log for JavaScript Batch Frameworks
```

Comment:
Imho this should be a subtopic under https://github.com/saucelabs/node-testcomposer/pull/12/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R19

Reply:
fixed 👍